src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components';
+import { getPeople } from '../api';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getNormalizedPeople } from '../helpers';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | []>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
to remove blinking when firstly opening the people page